configuration/programs/programs.csv

```diff
@@ -3,7 +3,7 @@ Uuid,Void/Retire,Program concept,Outcomes concept,_order:1001
 7e76873c-ca9f-403c-9318-fd981e256149,,Urgent follow up,63600df3-1525-40b1-8336-e312f649c68f,
 209154ad-a651-4a87-ad3e-153b29e86eac,,Physical Therapy,63600df3-1525-40b1-8336-e312f649c68f,
 9e4d6bd5-28d2-4966-9ef4-cd5cf32588ba,,Cita Abierta,63600df3-1525-40b1-8336-e312f649c68f,
-8647de5e-a5fa-4d9d-ba4d-8351830c2d6b,,Pre-operative clearance,63600df3-1525-40b1-8336-e312f649c68f,
+8647de5e-a5fa-4d9d-ba4d-8351830c2d6b,TRUE,Pre-operative clearance,63600df3-1525-40b1-8336-e312f649c68f,

```

Comment:
@basrem  Create a new line altogether to do the voiding and leave the previous line.